nuxeo-vision-aws/src/test/java/org/nuxeo/vision/aws/test/TestAmazonRekognitionProvider.java

```diff
@@ -120,6 +122,6 @@ protected AmazonRekognitionProvider getProvider() {
     }
 
     protected boolean areCredentialsSet() {
-        return getProvider().getNativeClient() != null;
+        return StringUtils.isNotBlank(System.getProperty(PROVIDES_AWS_CREDENTIALS));
```

Comment:
`return Boolean.getProperty(PROVIDES_AWS_CREDENTIALS)` to allow using true/false in the config.

Reply:
:+1:  `Boolean.getBoolean` used.